docs/guide/tutorials/connect/tls.txt

```diff
@@ -1,3 +1,192 @@
 =======
 TLS/SSL
 =======
+
+The Node.js driver supports TLS/SSL connections to MongoDB that support TLS/SSL support.
+
+No Certificate Validation
+-------------------------
+
+If the MongoDB instance does not perform any validation of the certificate chain, include the ``ssl=true`` in the :manual:`URI ConnectionString </reference/connection-string/>` .
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+
+   const client = new MongoClient('mongodb://localhost:27017?ssl=true');
+
+   client.connect().then(async function() {
+     await client.close();
+   });
+
+Validate Server Certificate
+---------------------------
+
+If the MongoDB instance presents a certificate, to validate the server's certificate, pass the following when creating a ``MongoClient``\ :
+
+* A :manual:`URI ConnectionString </reference/connection-string/>` that includes ``ssl=true`` setting,
+
+* A connections options with the certificate for the Certificate Authority (\ ``sslCA``\ ) and the ``sslValidate`` setting set to ``true``
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+   const fs = require('fs');
+
+   // Read the certificate authority
+   const sslCA = [fs.readFileSync(__dirname + "/ssl/ca.pem")];
```

Comment:
should there be array brackets around fs.readFileSync here on line 37?

Reply:
It doesn't *have* to, but it's not incorrect. This gets directly passed to the `ca` option [here](https://nodejs.org/dist/latest-v10.x/docs/api/tls.html#tls_tls_createsecurecontext_options), which can be `string|string[]|Buffer|Buffer[]`